include/mbedtls/x509.h

```diff
@@ -378,7 +379,10 @@ int mbedtls_x509_time_is_future(const mbedtls_x509_time *from);
 
 /**
  * \brief          This function parses an item in the SubjectAlternativeNames
- *                 extension.
+ *                 extension. Please note that this function might allocate
+ *                 additional memory for a subject alternative name, thus
+ *                 mbedtls_x509_free_subject_alt_name has to be called
+ *                 to dispose of the structure afterwards.
```

Comment:
```suggestion
 *                 to dispose of this additional memory afterwards.
```
Just makes it clearer that we don't free the SAN struct itself.